src/prescriptionData.js

```diff
@@ -0,0 +1,27 @@
+export const prescriptions = {
+  thinkingTools: {
+    oblique: [],
+    research: [],
+    sciency: [],
+    softy: []
+  },
+  randR:{
+    rest: [],
+    calm: [],
+    movement: [],
+    nourishment: []
+  },
+  esteem: {
+    cliche: [],
+    curious: [],
+    selfAdministered: [],
+  },
+  laughter: {
+
+  },
+  toughLove: {
+    fromDuck: [],
+    fromTube: [],
+    selfAdministered: []
+  }
+}
```

Comment:
Still marinating on how to organize the "advice" options inside the categories structured here. Ideally, different kinds of action steps are delivered quite differently. e.g. an affirmation from the Affirmations API vs a youtube video of a stretch routine vs a thinking tool -- do I designate the "delivery method" in this data structure, or build in conditional functionality elsewhere?